spec/haproxy/templates/drain_spec.rb

```diff
@@ -31,6 +31,13 @@
           fi
 
           pid="$(cat ${pidfile})"
+          if ! [ pgrep -F "${pidfile}" ]; then
+            # In case haproxy_wrapper process is stale, pid_exists will be empty,
```

Comment:
There is no variable named `pid_exists`, like this, the comment is not very useful to me